iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/setup/iothub/DeviceTwinCommon.java

```diff
@@ -48,9 +48,9 @@
 {
     // Max time to wait to see it on Hub
     protected static final long PERIODIC_WAIT_TIME_FOR_VERIFICATION = 1000; // 1 sec
-    protected static final long MAX_WAIT_TIME_FOR_VERIFICATION = 180000; // 180 sec
```

Comment:
This timeout was simply too long. Any test timeout failures will be caught in the first 30 seconds

Reply:
Do we have any long running tests? Token refresh tests? Connection status change tests?

Reply:
ping on this question